<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleView</class>
 <widget class="QMainWindow" name="SimpleView">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>3D Registration Toolkit</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="Widget_Main">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="Layout_Main">
    <item row="0" column="0">
     <widget class="QGroupBox" name="GroupBox_View">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>View</string>
      </property>
      <layout class="QGridLayout" name="Layout_View">
       <item row="0" column="0">
        <widget class="QVTKWidget" name="VTKWidget_Render" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="TabWidget_Process">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Widget_Initial">
       <attribute name="title">
        <string>Initial</string>
       </attribute>
       <layout class="QGridLayout" name="Layout_Initial">
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="GroupBox_Transformation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Transformation</string>
          </property>
          <layout class="QGridLayout" name="Layout_Transformation">
           <item row="1" column="1">
            <widget class="QLineEdit" name="LineEdit_RotationY"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Label_RotationX">
             <property name="text">
              <string>Rotation X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="LineEdit_RotationX"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="Label_RotationY">
             <property name="text">
              <string>Rotation Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="Label_RotationZ">
             <property name="text">
              <string>Rotation Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="LineEdit_RotationZ"/>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="LineEdit_TranslationZ"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="Label_TranslationZ">
             <property name="text">
              <string>Translation Z</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="Label_TranslationY">
             <property name="text">
              <string>Translation Y</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="Label_TranslationX">
             <property name="text">
              <string>Translation X</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="LineEdit_TranslationX"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="LineEdit_TranslationY"/>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="PushButton_Transform">
             <property name="text">
              <string>Transform</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QPushButton" name="PushButton_RandomT">
             <property name="text">
              <string>Random</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="GroupBox_ModelChoice">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Model</string>
          </property>
          <layout class="QGridLayout" name="Layout_ModelChoice">
           <item row="0" column="0">
            <widget class="QRadioButton" name="RadioButton_Source">
             <property name="text">
              <string>Source Model</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="RadioButton_Target">
             <property name="text">
              <string>Target Model</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="GroupBox_Normalization">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Normalization</string>
          </property>
          <layout class="QGridLayout" name="Layout_Normalization">
           <item row="0" column="1">
            <widget class="QLineEdit" name="LineEdit_Bounding"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Label_Bounding">
             <property name="text">
              <string>Bounding Box</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="PushButton_Normalization">
             <property name="text">
              <string>Normalization</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="GroupBox_Noising">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Noising</string>
          </property>
          <layout class="QGridLayout" name="GridLayout_Noising">
           <item row="0" column="1">
            <widget class="QLineEdit" name="LineEdit_Sigma">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Label_Sigma">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Sigma</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="PushButton_Noising">
             <property name="text">
              <string>Noising</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Widget_Register">
       <attribute name="title">
        <string>Register</string>
       </attribute>
       <layout class="QGridLayout" name="Layout_Register">
        <item row="1" column="0">
         <widget class="QGroupBox" name="GroupBox_SamplingDensity">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Sampling Density</string>
          </property>
          <layout class="QGridLayout" name="Layout_SamplingDensity">
           <item row="0" column="1">
            <widget class="QLineEdit" name="LineEdit_SourceSD"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="LineEdit_TargetSD"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Label_SourceSD">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Source Model</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" rowspan="2">
            <widget class="QLabel" name="Label_TargetSD">
             <property name="text">
              <string>Target Model</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="PushButton_LMICP">
          <property name="text">
           <string>LMICP</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="PushButton_ConvexHull">
          <property name="text">
           <string>Convex Hull Extraction</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="PushButton_3PCHM">
          <property name="text">
           <string>3PCHM</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Widget_View">
       <attribute name="title">
        <string>View</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="GroupBox_ViewModel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>ViewModel</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QCheckBox" name="ViewCheck_Source">
             <property name="text">
              <string>Source</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="ViewCheck_Target">
             <property name="text">
              <string>Target</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="LineEdit_SourceB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="LineEdit_SourceG">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="LineEdit_SourceR">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="LineEdit_TargetR">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="LineEdit_TargetG">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="LineEdit_TargetB">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="PushButton_Render">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>100</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Render</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777214</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>ViewType</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="4" column="1" colspan="3">
            <widget class="QCheckBox" name="ViewCheck_PointData">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>PointData</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="Label_PolyDataOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QCheckBox" name="ViewCheck_Axis">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Axis</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2" colspan="2">
            <widget class="QLineEdit" name="LineEdit_PointDataOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="7" column="2" colspan="2">
            <widget class="QLineEdit" name="LineEdit_PolyDataOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="Label_PointDataOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1" colspan="3">
            <widget class="QCheckBox" name="ViewCheck_ConvexHull">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>ConvexHullData</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="3">
            <widget class="QCheckBox" name="ViewCheck_PolyData">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>PolyData</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="Label_ConvexHullOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Opacity</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2" colspan="2">
            <widget class="QLineEdit" name="LineEdit_ConvexHullOpacity">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="Action_OpenSource"/>
    <addaction name="Action_OpenTarget"/>
    <addaction name="separator"/>
    <addaction name="Action_Exit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Action_OpenSource"/>
   <addaction name="Action_OpenTarget"/>
  </widget>
  <action name="Action_OpenSource">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Source Model</string>
   </property>
   <property name="iconText">
    <string>Source Model</string>
   </property>
   <property name="toolTip">
    <string>Source Model</string>
   </property>
  </action>
  <action name="Action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Action_OpenTarget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Target Model</string>
   </property>
   <property name="iconText">
    <string>Target Model</string>
   </property>
   <property name="toolTip">
    <string>Target Model</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
